### ISSUE TITLE
NEWS: GENERAL!

### ISSUE THREAD

**Issue by fjperini** [OSGeo/homebrew-osgeo4mac#790]
We are renaming the formulas to avoid problems when you have another version of the same package installed.

We recommend that you wait a few hours and that you previously remove these dependencies
```
brew uninstall --ignore-dependencies sip sip-qt5 pyqt pyqt-qt5 qscintilla2 qscintilla2-qt5 pyqt5-webkit gdal gdal2 postgis postgis2 liblas liblas-gdal2
```

among others (once the process is finished we will inform).

And then do
```
brew tap-pin osgeo/osgeo4mac
```
We regret the inconvenience, but we believe that it will be the best for the future.

We need to use our versions and that the user does not have another installed.

**Comment by jordan-heemskerk** [comment]
Thanks for this notice, was running into it right now and couldn't figure out what was happening!

**Comment by jordan-heemskerk** [comment]
@fjperini any update?

**Comment by fjperini** [comment]
@jordan-heemskerk generating the bottles, it will be ready in a couple of hours.

**Comment by fjperini** [comment]
@jordan-heemskerk Given that the process is too slow and unexpected problems arose, we will do so later.
We regret the moleesties, they can install everything as they came doing it.

**Comment by fjperini** [comment]
In case you did the above, you can restore with:
```

brew tap-unpin osgeo/osgeo4mac

brew reinstall sip-qt5 pyqt-qt5 qscintilla2-qt5 pyqt5-webkit gdal2 postgis2 liblas-gdal2

brew unlink sip-qt5 pyqt-qt5 qscintilla2-qt5 pyqt5-webkit gdal2 postgis2 liblas-gdal2 && brew link sip-qt5 pyqt-qt5 qscintilla2-qt5 pyqt5-webkit gdal2 postgis2 liblas-gdal2
```

**Comment by fjperini** [comment]
Applying new changes, wait for this to merge https://github.com/OSGeo/homebrew-osgeo4mac/pull/806

**Comment by vizowl** [comment]
I also having trouble with insighttoolkit - which is in both osgeo4mac and brewsci/science - is there a plan to rename this too?

**Comment by fjperini** [comment]
@vizowl Yes, we will soon.

My advice is to uninstall or remove that tap, and then install our version.

**Comment by fjperini** [comment]
that there are duplicate formulas is a great inconvenience, but we need ours since the ones of core or of other taps, they are outdated, without optional dependencies required or already without support.

**Comment by miguelangelperg** [comment]
Hi @fjperini 
All installed from osgeo4mac tap except *R* and *cairo* (from r_sethrfore) Other taps in my /Homebrew/Library/:  
- /brewsci/homebrew-bio/
- /brewsci/homebrew-science/
- /homebrew/

A partial picture of my situation. I would really appreciate your help

![upgrade_pg3](https://user-images.githubusercontent.com/42784270/53955965-a2ce7b80-40da-11e9-9476-8eadc2f40a5e.jpg)

**Comment by luispuerto** [comment]
@miguelangelperg please can you put this info in the new issue I mentioned here https://github.com/OSGeo/homebrew-osgeo4mac/issues/769#issuecomment-470514424 ?? I think it's wiser if you concentrate al the info related to your problem in just one issue, if you think it's related.
